src/display/annotation_layer.js

```diff
@@ -331,9 +331,13 @@ class AnnotationElement {
   get _commonActions() {
     const setColor = (jsName, styleName, event) => {
       const color = event.detail[jsName];
-      event.target.style[styleName] = ColorConverters[`${color[0]}_HTML`](
-        color.slice(1)
-      );
+      const colorType = color[0];
+      const colorArray = color.slice(1);
+      event.target.style[styleName] =
+        ColorConverters[`${colorType}_HTML`](colorArray);
+      this.annotationStorage.setValue(this.data.id, {
+        [styleName]: ColorConverters[`${colorType}_rgb`](colorArray),
+      });
```

Comment:
I'm probably missing some context, so possibly a stupid question, but any chance that we could just store the "HTML"-format instead and avoid having to add a bunch of new methods in the `ColorConverters` class?

Reply:
We get the value as it has been set in a field property and we must keep it like this because a script could use it later.
We could simplify that stuff in just converting the html color into a RGB array, I added this code in order to have only one conversion.